examples/ggui_examples/mass_spring_game_ggui.py

```diff
@@ -26,6 +26,9 @@
 rest_length = ti.field(dtype=ti.f32,
                        shape=(max_num_particles, max_num_particles))
 
+# gray color
+gray = (128, 128, 128, 255)
```

Comment:
All the other GGUI APIs use floats between 0 and 1 to represent color. The `text_colored` API should do the same for consistency.

Reply:
There's a chance that IMGUI automatically guesses the color range used (see the existing `color_edit_3` API) so perhaps nothing needs to be done here. But please investigate and confirm this.

Reply:
OK I will double check it.